main/com.spotify/manifest.tt

```diff
@@ -216,7 +216,7 @@ class @com.spotify
   #[doc="search for playlists on Spotify"];
 
   list query show extends playable(out id: Entity(com.spotify:show)
-                                        #[string_values="tt:show"]
+                                        //#[string_values="tt:show"]
```

Comment:
why is this change?

Reply:
The tt:show dataset doesn't actually exist. It's fine on CLI / Kubeflow because we ignore it, but when I upload the device to Thingpedia the server complains.